app/controllers/api/v2/providers_controller.rb

```diff
@@ -77,8 +77,27 @@ def sync_courses_with_search_and_compare
         head :ok
       end
 
+      def suggest
```

Comment:
Are we sticking with `suggest`? I think I prefer `search`

Reply:
I feel that `suggest` describes the intended usecase better, and regardless it's the terminology I've already used across both PRs and I think it's much more hassle than its worth to change that now unless we decide it's a significant enough hindrance.